### ISSUE TITLE
Add code quality tools

### ISSUE THREAD

**Issue by lejoe** [ginetta/skeleton#9]
I removed all lints for now.
We should research which ones we want to use
- jslint (js)
- eslint (js)
- or [grunt-jscs](https://github.com/jscs-dev/grunt-jscs/)
- scss (sass)
- plato (js)
- phantomas (css, pageload, compression)
- [grunt-devperf](https://github.com/gmetais/)

**Comment by peterbraden** [comment]
+1 for eslint. (and use es6!!)

**Comment by lejoe** [comment]
http://stylelint.io/

**Comment by meodai** [comment]
@lejoe we should use a SASS linter, not a CSS linter. since the output is easily configurable.

**Comment by lucalanca** [comment]
agreed with @meodai. My only problem is that I only know (scss-lint)[https://github.com/brigade/scss-lint] which is ruby (yikes!) and meant with ruby-sass in mind (yikes!).
